rest/blip_api_attachment_test.go

```diff
@@ -345,6 +347,9 @@ func TestPutAttachmentViaBlipGetViaRest(t *testing.T) {
 	getAttachmentRequest.SetProfile(db.MessageGetAttachment)
 	getAttachmentRequest.Properties[db.GetAttachmentDigest] = input.attachmentDigest
 	getAttachmentRequest.Properties[db.GetAttachmentID] = input.docId
+	if !base.IsDefaultCollection(collection.ScopeName(), collection.Name()) {
```

Comment:
use `bt.addCollectionProperty(getAttachmentRequest)` here so we can make changes in a central location